src/replace_landsurface_with_ERA5land_IC.py

```diff
@@ -220,9 +221,9 @@ def swap_land_era5land(mask_fullpath, ic_file_fullpath, ic_date):
 
     # Find one "swvl1" file in the archive and create a generic filename
     ERA_FIELDN = 'swvl1'
-    land_yes = ERA_DIR + ERA_FIELDN + '/' + yyyy + '/'
-    era_files = glob(land_yes + '/' + ERA_FIELDN + '*' + yyyy + mm + '*nc')
-    era5_fname = land_yes + '/' + era_files[0].split('/')[-1]
+    land_yes = os.path.join(ERA_DIR, ERA_FIELDN, yyyy)
+    era_files = glob(os.path.join(land_yes, ERA_FIELDN + '*' + yyyy + mm + '*nc'))
+    era5_fname = os.path.join(land_yes, os.path.basename(era_files[0]))
```

Comment:
```suggestion
    era5_fname = era_files[0]
```

See comment above about redundancy.